app/models/consultee.rb

```diff
@@ -102,6 +102,10 @@ def application_link
     end
   end
 
+  def can_resend_magic_link?
+    magic_link_last_sent_at.nil? || magic_link_last_sent_at <= 1.minute.ago
```

Comment:
This logic is confusing. I think it's correct but I had to double-take: here `<=` actually means _before_ one minute ago, which is correct, but it looks like it's saying _less than_ one minute ago, which is the opposite. Don't know if there's any way around that.

Reply:
How about splitting the method in more methods?

```ruby 
def can_resend_magic_link?
    magic_link_last_sent_at_nil? || magic_link_last_sent_at_less_than_1_min?
end

def magic_link_last_sent_at_nil?
    magic_link_last_sent_at.nil?
end

def magic_link_last_sent_at_less_than_1_min?
    magic_link_last_sent_at <= 1.minute.ago
end
```